ui/src/app/components/resource-list.spec.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { MemoryRouter } from 'react-router';
+import { mount } from 'enzyme';
+
+import { WorkspaceResource } from 'generated/fetch';
+
+import { waitOneTickAndUpdate } from 'testing/react-test-helpers';
+import { stubResource } from 'testing/stubs/resources-stub';
+import { workspaceDataStub } from 'testing/stubs/workspaces';
+
+import { ResourceList } from './resource-list';
+
+export const RESOURCE_TYPE_COLUMN_NUMBER = 1;
```

Comment:
I would like us to move away from the ALL_CAPS convention. Most of our variables should be constants these days and those that are not should be narrowly scoped so as not to need special treatment. No need to adjust in this PR, I'm just using it to spark the conversation.